frontend/packages/dev-console/src/components/helm/HelmInstallUpgradePage.tsx

```diff
@@ -18,9 +19,10 @@ import {
   HelmActionOrigins,
 } from './helm-types';
 import { getHelmActionValidationSchema } from './helm-validation-utils';
-import { getHelmActionConfig, getChartValuesYAML } from './helm-utils';
+import { getHelmActionConfig, getChartValuesYAML, getHelmChartReadme } from './helm-utils';
 import NamespacedPage, { NamespacedPageVariants } from '../NamespacedPage';
 import HelmInstallUpgradeForm from './form/HelmInstallUpgradeForm';
+import './helm.scss';
```

Comment:
Name of the `scss` file should be same as the component file name. Also, you don't need to load the css in this file. It should be loaded in the actual modal component.